Makefile

```diff
@@ -16,6 +16,7 @@ CERT_MANAGER_CHART_VERSION := 1.11.0
 
 .PHONY: lint
 lint:
+	go mod vendor
```

Comment:
We have to make sure `gogoproto` exists in vendor directory, since we use `gogoproto`.

Reply:
I split into `lint-go` and `lint-proto` 🙂